packages/map-template/src/components/Search/Search.jsx

```diff
@@ -322,20 +368,38 @@ function Search({ onSetSize }) {
                 category={selectedCategory}
                 disabled={searchDisabled} // Disabled initially to prevent content jumping when clicking and changing sheet size.
             />
-            <div className="search__scrollable prevent-scroll" {...scrollableContentSwipePrevent}>
-                {categories.length > 0 &&
-                    <div ref={categoriesListRef} className="search__categories">
-                        {categories?.map(([category, categoryInfo]) =>
-                            <mi-chip
-                                icon={categoryInfo.iconUrl}
-                                background-color={primaryColor}
-                                content={categoryInfo.displayName}
-                                active={selectedCategory === category}
-                                onClick={() => categoryClicked(category)}
-                                key={category}>
-                            </mi-chip>
-                        )}
-                    </div>}
+            <div className="prevent-scroll" {...scrollableContentSwipePrevent}>
+                <div className="search__scrollable">
```

Comment:
Can we extract all JSX and logic related to the categories into a separate component? I think that would be a clear separation of concerns.

Reply:
Just the categories or the categories with scroll buttons as well?

Reply:
Great idea, refactored the code and extracted the categories into its own component. Now the search component has been simplified a lot.